.github/workflows/release.yml

```
@@ -13,256 +13,270 @@
-          needs.deploy-simulation.result == 'success'
+      # --- S3 real upload ---
+      - name: Configure AWS credentials (OIDC)
+        if: ${{ steps.flags.outputs.aws == 'true' }}
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Upload to S3
+        if: ${{ steps.flags.outputs.aws == 'true' }}
+        run: |
```

actionlint (1.7.7)

property "bump-and-tag" is not defined in object type {build-and-release: {outputs: {}; result: string}}

(expression)